src/Services/CsvToTable.php

```diff
@@ -237,13 +241,6 @@ public function exec()
                 if ($errCount >= $this->maxRecErr) {
                     $this->error("Failed %d times. Halting further processing", $errCount);
                     $this->feeder->flush();
-                    $this->error(
-                        "%s(%d): %s\n%s",
-                        $except->getFile(),
-                        $except->getLine(),
-                        $except->getMessage(),
-                        $except->getTraceAsString()
-                    );
                     throw $except;
```

Comment:
This is context leakage: There is no need to write this error/exception to log. It's handled by laravel